biom/parse.py

```diff
@@ -245,6 +246,98 @@ def get_axis_indices(biom_str, to_keep, axis):
     return idxs, json.dumps(subset)[1:-1]  # trim off { and }
 
 
+def parse_uc(fh):
+    """ Create a Table object from a uclust/usearch/vsearch uc file.
+
+        Parameters
+        ----------
+        fh : file handle
+            The ``.uc`` file to be parsed.
+
+        Returns
+        -------
+        biom.Table : The resulting BIOM table.
+
+        Raises
+        ------
+        ValueError
+            If a sequence identifier is encountered that doesn't have at least
+            one underscore in it (see Notes).
+
+        Notes
+        -----
+        This file assumes that sequence identifiers in this file are in QIIME's
+        "post-split-libraries" format, where the identifiers are of the form
+        ``<sample-id>_<sequence-id>``. Everything before the first underscore
+        will be used as the sample identifier in the resulting ``Table``.
+        The information after the first underscore is not used directly, though
+        the full identifiers of seeds will be used as the observation
+        identifier in the resulting ``Table``.
+
```

Comment:
I thought this was going to capture stats on the hits as well too? Or is that substantially complicated that it isn't worth it?